content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,6 +38,45 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 {{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
 
+## Guidelines for contributing content
+
+Kubernetes documentation only hosts content from projects within the
+ `kubernetes` or `kubernetes-sigs` GitHub organizations or from current CNCF
+  projects ([Graduated/Incubating](https://www.cncf.io/projects/), [Sandbox](https://www.cncf.io/sandbox-projects/)).
+The CNCF doesn't maintain a list of archived projects. However, you can
+ search the CNCF Technical Oversight Committee GitHub 
+ [pull requests](https://github.com/cncf/toc/labels/archive) to find
+  proposals that are for archiving projects.
```

Comment:
We should propose that the CNCF publish a list of archived projects (and the dates the project was active and then archived). It would be useful for things like this.

Reply:
The CNCF will list archived projects online, according to the [Project Archiving Process](https://github.com/cncf/toc/blob/master/process/archiving.md). I sent an [email](https://lists.cncf.io/g/cncf-toc/message/3539) to the CNCF TOC mailing list asking about the time frame for putting the list online and where the list will be located.